nerves-hub/setup/firmware.md

```diff
@@ -20,3 +20,25 @@ Firmware can also be signed while publishing:
 mix nerves_hub.firmware publish --key devkey
 ```
 
+## Delta Updates
+
+With Delta updates, NervesHub can compare two binary files and only update the pieces of the firmware that have changed. This can reduce the size and cost of OTA updates by up to 95%. In order use Delta Updates, you need to do the following:
```

Comment:
```suggestion
With Delta updates, NervesHub can compare two binary files and only update the pieces of the firmware that have changed. This can reduce the size and cost of OTA updates by up to 95%. There are a few requirements to use Delta Updates:
```